.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,116 @@
+name: "Bug report"
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value:  |
+        ## READ BEFORE OPENING ISSUES
+        All bug reports require you to **USE CANARY BUILDS**. Please include the version name and version code in the bug report.
+        **DO NOT** open issues regarding root detection.
+        **DO NOT** ask for instructions.
+        **DO NOT** report issues if you have any modules installed.
+        Without following the rules above, your issue will be closed without explanation.
+  - type: checkboxes
+    id: preliminary_checklist
+    attributes:
+      label: Preliminary checklist
+      options:
+        - label: "I have read the [README](https://github.com/topjohnwu/Magisk/blob/master/README.MD)."
+          required: true
+        - label: "I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue."
+          required: true
+        - label: "I have updated Magisk Canary to the latest version. The bug is reproducible on this latest version."
+          required: true
+  - type: dropdown
+    id: device_architecture
+    attributes:
+      label: Device architecture
+      multiple: false
+      options:
+        - arm
+        - arm64
+        - arm64-v8a
+        - armeabi-v7a
+        - x86
+        - x64
+    validations:
+      required: true
+  - type: input
+    id: device_model
+    attributes:
+      label: Device model
+      description: What is your device model? This can usually be found in your device's settings in the "About" section. Also do mention if you're using a Custom ROM.
+      placeholder: (e.g. `Nokia 6.1 Plus (DRG_Sprout)`, or `SM-G960UZKABST`, etc.)
+    validations:
+      required: true
+  - type: dropdown
+    id: android_version
+    attributes:
+      label: Android version
+      description: What version of Android are you runing currently?
+      options:
+        - 13 (Developer Preview)
+        - 12
+        - 11
+        - 10
+        - 9
+        - 8.1
+        - 8.0
+        - 7.1
+        - 7.0
+        - 6
+        - 5.1
+        - 5.0
+        - 4.4
+        - 4.3
+        - 4.2
+        - 4.1
```

Comment:
I think we can remove 4.1~4.4 as Magisk only supports Android 5.0+ since v23.

Reply:
Sure I'll do those if required.